nfs-ganesha/build/build_rpm

```diff
@@ -117,12 +117,12 @@ if [ "$THROWAWAY" = false ] ; then
 }
 EOF
     # post the json to repo-extra json to chacra
-    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}/repos/${chacra_endpoint}/extra/
+    curl -X POST -H "Content-Type:application/json" --data "@$WORKSPACE/repo-extra.json" -u $CHACRACTL_USER:$CHACRACTL_KEY ${chacra_url}/repos/${chacra_endpoint}/extra/flavors/${FLAVOR}
```

Comment:
yes, but it needs to come before ``/extra``.   ``ceph-dev-build`` is the best job to look at for examples of chacra integration, see these couple lines here:

https://github.com/ceph/ceph-build/blob/master/ceph-dev-build/build/build_rpm#L174-L176